packages/SketchMorph2-Core.package/M2Backend.class/instance/prepareTrianglePenSource.st

```diff
@@ -1,7 +1,9 @@
 drawing
 prepareTrianglePenSource
 	| triangleForm temporaryPen |
-	triangleForm := Form extent: self brushSize asPoint.
+	triangleForm := Form
```

Comment:
I'm not sure the style is correct here since the following is not a cascade.

Reply:
Like so?

```smalltalk
self pen sourceForm:
	(Form extent: self rectangleBrushSize
		offset: (self penNibOffset: self rectangleBrushSize))
		fillBlack.
```